### DISCUSSION TITLE
How many days from 🎃 to 🎄?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#831] [Galaxy Trivia] [answered]
**Bonus:** What holiday is *approximately* XX days from today?

**Comment by peterbaccaro** [comment]
The number of days between Halloween and Christmas is fixed every year. There are always **55 days** between Halloween (October 31st) and Christmas (December 25th).

Today is 28th May 2023.  20 days from now is 17th June 2023. 
**Juneteenth** on the 19th of June is a federal holiday in the United States commemorating the emancipation of enslaved African Americans. Freedom finally came on June 19, 1865, when some 2,000 Union troops arrived in Galveston Bay, Texas. The army announced that the more than 250,000 enslaved black people in the state, were free by executive decree. This day came to be known as "Juneteenth," by the newly freed people in Texas.

**Comment by SANKARAMDAS** [comment]
In both old and new calendars Western Christmas(December 25th) and Hallowe'en(October 31st) are 55 days apart.

**Comment by Carol42** [comment]
As a programmer, I might get a bit confused with this question 😆 

![269918384_158891039803049_3237642415194922773_n](https://github.com/ParanoidUser/yolo/assets/63017741/b7c9179b-101c-4465-9246-31ff141e2fc5)

**Comment by anjutares** [comment]
From Halloween, which is on October 31st, to Christmas, which is on December 25th, there are 55 days.

If today is May 28th, a holiday that is approximately 55 days from today would be:
July 22nd (depending on the year, it could be different, so please note that this date is an example based on the current year).

**Reply by ParanoidUser** [reply]
Any special holiday on that date? 🤭

**Reply by anjutares** [reply]
Hahahaha yes, near that day is my saint, San Juan (Saint John).

Here in Spain it is celebrated on June 24, but on the eve, that is to say the 23rd, all the people go out to celebrate it and they usually make bonfires on the beach and one of the traditions is to jump over the bonfire, something dangerous because people usually drink a lot that night hahaha.

Another festivity with a closer date is the festivity of Carmen, but they are less famous.
The Fiestas del Carmen are celebrations that take place in several coastal places in Spain (I'm from Valencia), in honor of the Virgen del Carmen , considered the patron saint of sailors and protector of sailors. These festivities usually take place in July, mainly around July 16. 
As always, religious acts, partying, drinking and fireworks hahahaha

**Comment by peterbaccaro** [comment]
The number of days between Halloween and Christmas is fixed every year. There are always **55 days** between Halloween (October 31st) and Christmas (December 25th).

Today is 28th May 2023. 55 days from now is 22nd July 2023.
**World Brain Day** is an annual observance organized by the World Federation of Neurology (WFN) to raise awareness about various neurological conditions and promote brain health. It takes place on July 22nd each year.
Also **school summer holliday's/break** starts around July 22nd each year.  In the UK it does anyway.

**Comment by ParanoidUser** [comment]
**Hint:** 22/7 = ?

**Reply by peterbaccaro** [reply, marked as answer]
Thanks for the clue, the specific answer you are looking for here is **Pi Approximation Day**.

Pi Approximation Day is observed on July 22 (22/7 in the day/month date format), since the fraction 22⁄7 is a common approximation of π, which is accurate to two decimal places and dates from Archimedes.

22/7 = 3.142857142857143

**Reply by ParanoidUser** [reply]
**Yay!** 👏 Why did pi have its driver's license revoked?
Because it didn't know when to stop.

**Comment by Carol42** [comment]
Between Halloween and Christmas, there are 'pumpkin π' days! 😄
